lib/fibonacci.rb

```diff
@@ -4,6 +4,28 @@
 # e.g. 1st fibonacci number is 1
 # ....
 # e.g. 6th fibonacci number is 8
+
+# Time complexity is O(n) as iteration happens only once
```

Comment:
The time complexity is correct. However, I'm having a hard time following your explanation. The loop iterates more than once. I recommend more practice with explaining your reasoning for the time and space complexities.